### PR TITLE
docs: Review docstrings

### PR DESCRIPTION
**Pull request by YurelyCamacho**
## Description

Review docstrings:
forecast_swiss.py
metrics.py
ngboost_models.py
preprocessing.py
epigraphhub_db.py
worldbank.py

Fixed docstring formatting as it was in most cases.

## Checklist

- [x] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/thegraphnetwork/epigraphhub_py/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I've read the [`CONTRIBUTING.md`](https://github.com/thegraphnetwork/epigraphhub_py/blob/master/CONTRIBUTING.md) guide.
- [ ] I've used pre-commit hook locally.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.
- [ ] Semantic-release prefix title used (case sensitive):
  - Use the prefix `chore(docs):` for Examples / docs / tutorials / dependencies update
  - Use the prefix `fix:` for a bug fix
    (non-breaking change which fixes an issue).
  - Use the prefix `chore(improvement):` for an improvement
    (non-breaking change which improves an existing feature).
  - Use the prefix `feat:` for a new feature
    (non-breaking change which adds functionality).
  - Use the prefix `BREAKING CHANGE:` for a breaking change
    (fix or feature that would cause existing functionality to change).
  - Use the prefix `chore(security):` for a security fix.

**Note**: The title prefix can be also expanded with more information
inside the parenthesis. Some examples for a full title (prefix + message):
- `fix(login): Fix the login page`.
- `feat(forecast): Add a weather forecast page`.
- `chore(security-login): Add a weather forecast page`.

### CODE REVIEW

File: epigraphhub/data/colombia/extract.py [hunk 1]
```diff
@@ -59,14 +61,17 @@ def _table_last_update() -> datetime:
 
 def _web_last_update() -> datetime:
     """
-    This method will request the maximum date found in Colombia data through Socrata API
-    and returns it as a datetime object for further evaluation.
+    This method will request the maximum date found in Colombia data through Socrata API and returns it as a datetime object for further evaluation.
```
Comment (xmnlab) [inline epigraphhub/data/colombia/extract.py:64]: please keep the line with less than 80 characters (<=79)

File: epigraphhub/data/colombia/extract.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 """
 Last change on 2022/09/22
-Comparing Colombia Governmental COVID data consists in a step before pushing it to
-the SQL Database. Is responsible for retrieving the last date in both CSV and SQL table.
+Comparing Colombia Governmental COVID data consists in a step before pushing it to the SQL Database. Is responsible for retrieving the last date in both CSV and SQL table.
```
[the remark this answers is not in the document]
Comment (YurelyCamacho) [reply epigraphhub/data/colombia/extract.py:3]: No, locally it does not appear like that.

File: epigraphhub/data/worldbank.py [hunk 3]
```diff
@@ -127,19 +135,25 @@ def search_in_database(keyword):
 
 
 def search_in_indicators(keyword, db=2):
-
+
```
Comment (xmnlab) [inline epigraphhub/data/worldbank.py:138]: I think we don't need this empty line here

File: epigraphhub/data/worldbank.py [hunk 4]
```diff
@@ -151,29 +165,36 @@ def search_in_indicators(keyword, db=2):
 
 
 def get_worldbank_data(ind, country, db=2, time="all", columns=None):
-
+
```
Comment (xmnlab) [inline epigraphhub/data/worldbank.py:168]: same here

**Review by xmnlab — COMMENTED** [review]
I've not read all the changes .. but in general .. it looks good to me. 
thanks for working on that.

@eduardocorrearaujo could you review this one?

**Comment by xmnlab** [comment]
Ping  @eduardocorrearaujo @fccoelho

**Review by fccoelho — APPROVED** [review]

**Comment by fccoelho** [comment]
It is looking good. We can merge it

**Review by eduardocorrearaujo — APPROVED** [review]
It looks good. Thank you @YurelyCamacho

**Pull request merged**